pages/linux/torsocks.md

```diff
@@ -0,0 +1,12 @@
+# torsocks
+
+> Use an external application through the Tor network.
+> More information: <https://github.com/dgoulet/torsocks>.
```

Comment:
```suggestion
> More information: <https://gitlab.torproject.org/tpo/core/torsocks/>.
```

As it says on the github page, upstream has been moved to https://gitlab.torproject.org/tpo/core/torsocks/ . Also, comparing the commits, gitlab has the newer commits:

https://github.com/dgoulet/torsocks/commits/master/
https://gitlab.torproject.org/tpo/core/torsocks/-/commits/main